S2VX.Game/S2VXGameBase.cs

```diff
@@ -18,6 +19,11 @@ public class S2VXGameBase : osu.Framework.Game {
 
         protected override Container<Drawable> Content { get; }
 
+        public override void SetHost(GameHost host) {
+            base.SetHost(host);
+            host.Window.CursorState |= CursorState.Hidden;
```

Comment:
I assume this hides the system cursor?

Reply:
Yeah

Reply:
Yeah. I can't really test this because the `Window` returns null when running pipeline tests.